docs/PLUGIN-DEV.md

```
@@ -283,3 +283,128 @@ docker compose up -d
+When a migration fails part-way through, the `_prisma_migrations` table retains a row for the failed migration with a null `finished_at` and a non-null `logs` field. `prisma migrate status` reports it as "Not finished."
+
+### 1. Identify the failure
+
+```bash
+cd packages/kanban-viewer
+npx prisma migrate status
+```
+
+Look for output like:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)